SCRUM-64

IlhamAlf90 opened the pull request:
index Footer till startsidan

IlhamAlf90 committed:
Create index.html

index <Footer> till startsidan

IlhamAlf90 committed:
Create style.css

style.css till footer

rfhager reviewed (changes requested):
Behöver uppdateras enligt de ändringar som diskuterades på reviewn.

- Ändring av plats på sociala medier samt adress
- Synka färgschema med headern

Utöver det så ser jag att det är satt en max-width på footern som gör att det ser lite tokigt ut, tanken var att den bara skulle gälla för allt innehåll på sidorna för att sakpa en enhetlighet för innehållet, förutom header och footer som bör löpa från sida till sida på skärmen.

Saknar png filerna för att få bilderna att fungera för ikonerna. Kan dessa kanske läggas till i en mapp som förslagsvis döps till img?

Ska denna CSS filen ligga rakt i repositoriet eller ska vi ha den i en mapp som heter CSS?

Ser att responsiviteten fungerar bra nu men att det blir ett stort tomrum längst ner i mobilvyn, kan man minska det?

IlhamAlf90 committed:
Update index.html

ändrade från <aside> till <section>

IlhamAlf90 committed:
Update style.css

ändrade lite i style så att den passar mobilskärm, ändrade också icons plats så att den hamnar sist till höger.

IlhamAlf90 committed:
Add files via upload

icons for the Footer

IlhamAlf90 committed:
Create kontakt.html

IlhamAlf90 committed:
Create kontakt.html

IlhamAlf90 committed:
Rename kontaktsidan to kontaktstyle.css

IlhamAlf90 committed:
Add files via upload

IlhamAlf90 committed:
Update index.html

ändring och förbättring på footer

IlhamAlf90 committed:
Update style.css

ändring och förbättring på Footer

karinEkenberg reviewed (commented):
Jag får inte fram bilderna i footern när jag tittar?

IlhamAlf90 commented:
jag ska försöka skicka de i en fil istället

IlhamAlf90 committed:
Add files via upload

IlhamAlf90 committed:
Update kontakt.html

ändringar och förbättringar på kontaktsidan

IlhamAlf90 committed:
Update kontaktstyle.css

ändringar och förbättringar på css filen till kontaktsidan
